SPECS/systemd/systemd.spec

```diff
@@ -108,6 +117,15 @@ EOF
 %patch14 -p1
 %patch15 -p1
 
+# Portablectl patches
+%patch100 -p1
```

Comment:
I suggest using `%autosetup` instead and removing all `%patch` directives.

Reply:
Systemd SPEC already using %patches hence using the same. To move to %autosetup we need more validation to confirm all patches applied correctly. Hence going with %patch.

Reply:
What validation do you have in mind? If a patch doesn't apply correctly the package will fail to build, so any breaks should be quickly visible.

Reply:
Clear Linux also does this and autospec successfully applies the patches. So does this really need to change?